src/uri/validator/urivalidator.rs

```diff
@@ -1145,11 +1146,7 @@ mod tests {
             name: "hartley".into(),
             ..Default::default()
         };
-        let resource = UResource {
-            name: "rpc".into(),
-            id: Some(19999),
-            ..Default::default()
-        };
+        let resource = UResourceBuilder::for_rpc_response();
```

Comment:
can we please keep the code that explicitly specifies the (valid) resource field values?